packages/orchestration/src/exos/chain-hub.js

```diff
@@ -225,7 +225,7 @@ const ChainHubI = M.interface('ChainHub', {
   ),
   getDenom: M.call(BrandShape).returns(M.or(M.string(), M.undefined())),
   makeChainAddress: M.call(M.string()).returns(CosmosChainAddressShape),
-  resolveAccountId: M.call(M.string()).returns(M.string()),
+  resolveAccountId: M.call(M.or(M.string(), M.record())).returns(M.string()),
```

Comment:
Shouldn't this use CosmosChainAddressShape ?

Reply:
Yes, that's an improvement.